web3.go

```diff
@@ -49,6 +52,75 @@ func WeiAsGwei(w *big.Int) string {
 	return new(big.Rat).SetFrac(w, weiPerGwei).FloatString(9)
 }
 
+func CallConstantFunction(ctx context.Context, client Client, myabi abi.ABI, address, functionName string, parameters ...interface{}) (interface{}, error) {
+	var out interface{}
+	switch myabi.Methods[functionName].Outputs[0].Type.T {
+	case abi.BoolTy:
+		out = new(bool)
+	case abi.UintTy:
+		out = new(big.Int)
+	default:
+		out = new(string)
```

Comment:
Is it safe to default to string? Or should we say "unsupported type" (or could we fill in the rest pretty easily?)